jekyll/_cci2/runner-faqs-adoc

```diff
@@ -0,0 +1,93 @@
+= CircleCI Runner FAQs
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page answers frequently asked questions for the CircleCI Runner product.
+
+toc::[]
+
+=== How do I get customers on to the preview?
```

Comment:
Seems odd to refer to customers? Is this and the item below for internal use only?

Reply:
fixed